src/extension.ts

```diff
@@ -10,7 +10,13 @@ interface Settings {
   catalogs: String[],
   logs: {},
   format: {},
-  fileAssociations: JSON[]
+  fileAssociations: JSON[],
+  serverCapabilities: {},
```

Comment:
why do you need to send serverCapabilities? There's no such preference

Reply:
Added that in by accident on a rebase